bedrock/privacy/templates/privacy/notices/firefox-cliqz-quantum.html

```diff
@@ -0,0 +1,72 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-pebbles.html" %}
+
+{% block page_title %}
+  {{ doc.body.section.h2.get_text() }}
+{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_privacy' %}
+{% endblock %}
+
+{% block body_id %}firefox-notice{% endblock %}
+
+{% block string_data %}
+  data-tabpanel-open-text="{{ _('Show All') }}"
+  data-tabpanel-close-text="{{ _('Hide All') }}"
+  data-choices-copy="{{ _('Choose how you want to share this data in Firefox.') }}"
+  data-choices-button="{{ _('Choose…') }}"
+{% endblock %}
+
+{% block content %}
+<article class="section-content" itemtype="http://schema.org/Article">
+
+{% set header = doc.body.section.extract() %}
+{% set lead_in = doc.body.section.extract() %}
+{% set footnote = doc.body.select('#footnote') %}
+{% if footnote %}
+    {% set footnote = footnote[0].extract() %}
+{% endif %}
+
+<header class="privacy-head">
+  <div class="privacy-head-wrapper">
+    <h1 itemprop="name">{{ header.select('h2 span[class="privacy-header-firefox"]')|join|safe }}
```

Comment:
Shouldn't the header read `Firefox + Cliqz Privacy Notice`? (like the page title)

Reply:
Doh! Indeed.